templates/repo/commits_list.tmpl

```diff
@@ -56,6 +56,7 @@
 							{{else}}
 								</span>
 							{{end}}
+							<span style="cursor: pointer;vertical-align: middle;" class="clipboard" data-original="{{$.i18n.Tr "repo.commits.copy_sha"}}" data-success="{{$.i18n.Tr "repo.commits.copy_sha_success"}}" data-error="{{$.i18n.Tr "repo.commits.copy_sha_error"}}" data-content="{{$.i18n.Tr "repo.commits.copy_sha"}}" data-variation="inverted tiny" data-clipboard-text="{{.ID}}">{{svg "octicon-clippy"}}</span>
```

Comment:
Please refrain from adding additional inline CSS.

Reply:
I made an improvement by removing the inline style and using the existing ones for better consistency. 
Also, I added it inside the <a> tag to get stacked better with the other icons. 
Because it's inside the <a> tag I had to add `onclick="event.preventDefault()"`.
Is that ok to commit it like that?

Reply:
No, inline scripts are also not appreciated. Make it an actual `<button>` and you won't need it while also being semantically correct.

Reply:
Unfortunately, I can't just add a button outside the <a> tag. 
For example, If we have it as a button outside the link I must create new styles for that button to fit correctly
![copy_button](https://user-images.githubusercontent.com/7754679/101249069-d1510e80-3725-11eb-86b1-e5890deb6e81.PNG)
If there is also the icon for a signed commit we have the below result with the three dots and also we have to create new styles for the content to fit correctly in the cell and rearrange the content with the proper order.
![button_and_other](https://user-images.githubusercontent.com/7754679/101249495-f0e83700-3725-11eb-8f31-46e4c531be17.PNG)
If we put it inside the link with the same styles that other icons have there, the icons are consistent but we have to prevent the redirection to commit details somehow.
![copy_inside](https://user-images.githubusercontent.com/7754679/101250223-2db42e00-3726-11eb-9b1b-73e401598fec.png)
 
Whatever it must be done I believe that the copy should be consistent with the other copy actions across the site.